src/main/webapp/WEB-INF/web.xml

```diff
@@ -18,7 +18,10 @@
         <url-pattern>delete-tutor-session</url-pattern>
         <url-pattern>history</url-pattern>
         <url-pattern>manage-availability</url-pattern>
+        <url-pattern>manage-groups</url-pattern>
```

Comment:
Because manage-groups/replies/posts also GET groups/posts/replies, viewers that are not logged in will not be able to view group stuff. Should we allow not logged in users to view groups?

Reply:
Good point. I think it might be good in terms of privacy to make sure that only logged in users can see the posts and also in terms of getting more users (since people would have more reasons to register). What do you think?

Reply:
Yeah, that makes sense.